src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void toDrawFigure() {
+        System.out.println("Figure: square, area: " + getArea() + " sq. units, "
```

Comment:
Use only English in messages/code. The 'sq. units' abbreviation may not be clear to all readers, consider using 'square units' for better clarity and consistency.